.github/workflows/pr-labeler.yml

```diff
@@ -1,7 +1,7 @@
 name: PR Labeler
 
 on:
-  pull_request:
+  pull_request_target:
```

Comment:
I noticed that the PR labeller check is skipped after this change. Does it mean it will not work for internal PRs anymore?